stix_shifter_modules/ibm_security_verify/stix_translation/query_constructor.py

```diff
@@ -13,37 +12,49 @@
 
 # Source and destination reference mapping for ip and mac addresses.
 # Change the keys to match the data source fields. The value array indicates the possible data type that can come into from field.
-REFERENCE_DATA_TYPES = {"ipaddr": ["ipv4", "ipv4_cidr", "ipv6", "ipv6_cidr"],
-                        "proxy_ip": ["ipv4", "ipv4_cidr"],
-                        }
-REFERENCE_FILTER_TYPE = ["user-account",
-                         "ipv4-addr", "domain-name", "x-oca-event"]
+REFERENCE_DATA_TYPES = {
+    "ipaddr": ["ipv4", "ipv4_cidr", "ipv6", "ipv6_cidr"],
+    "proxy_ip": ["ipv4", "ipv4_cidr"],
+}
+REFERENCE_FILTER_TYPE = ["user-account", "ipv4-addr", "domain-name", "x-oca-event"]
 REFERENCE_EXCLUDE_FILTER_TYPE = ["category"]
 logger = logging.getLogger(__name__)
 
 
 class QueryStringPatternTranslator:
     QUERIES = []
- 
+    comparator_lookup = {
```

Comment:
This is not where comparators are defined now, it should be in a separate operators.json file. Are you basing this PR off of an older version?

Reply:
Hi , PR  is to revert the lookup changes because  CP4S 1.9 is not yet support the latest version stix-shifter and when the 1.10 will be released , I will make the required changes to support.